Source/utils/paths.cpp

```diff
@@ -86,6 +89,8 @@ const std::string &PrefPath()
 		prefPath = FromSDL(IOSGetPrefPath());
 #elif defined(NXDK)
 		prefPath = NxdkGetPrefPath();
+#elif defined(__amigaos__)
+		prefPath = std::string("PROGDIR:");

```

Comment:
Since you made SDL_GetPrefPath return this value is this overwrite really needed?

Reply:
Right, haven't noticed SDL_GetPrefPath there.

Reply:
But because of `prefPath = std::string("." DIRECTORY_SEPARATOR_STR);`  it is still needed. 
"./" won't work on Amiga.

Reply:
In that case preferably have the ifdef around the `if (FileExistsAndIsWriteable... }` block in that case.

Reply:
Done.